src/utils/analytics.ts

```diff
@@ -124,7 +130,7 @@ export function flushEvents() {
   return segment.flush();
```

Comment:
`rudderstack` does not support the `flush` method.

In order to reset information related to a previous user, you must use `.reset`

See [here](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-javascript-sdk/supported-api/#reset)